content/realtime/channel-lifecycle.textile

```diff
@@ -0,0 +1,141 @@
+---
+title: Channel Metadata and Lifecycle Events
+section: realtime
+index: 15
+jump_to:
+  Help with:
+    - Overview#overview
+    - Metachannels#metachannels
+    - Lifecycle Events#lifecycle-events
+    - Reactor Rules#reactor
+    - Permissions#permissions
+  Channel API Types:
+    - ChannelDetails#channel-details
+    - ChannelStatus#channel-status
+    - Occupancy#occupancy
+---
+
+h2(#overview). Overview
+
+The realtime "channel lifecycle":#lifecycle-events and "occupancy":#occupancy events are published onto "metachannels":#metachannels that can be subscribed to using our existing "realtime libraries":/realtime. This is often used in conjunction with the "Ably Reactor":/reactor, which allows you to send this data to your "serverless endpoints":/general/functions, "queues":general/queues, and "streams":general/firehose.
+
+The "REST API":/rest-api can also be used "enumerate all active channels":/rest/channel-lifecycle#enumeration-rest, or obtain the status of an "individual channel":#metadata-rest. At present, REST "requests":/rest/usage#request for lifecycle events will not count towards your message limit. If you are making use of "Reactor":/reactor however, then each message sent over Reactor *will* count towards your message limits.
```

Comment:
Perhaps:

You can also make use of the "REST API":/rest-api, to enumerate all active channels..